hooks/post_gen_project.py

```diff
@@ -47,15 +47,12 @@ def process_licence(licence_name):
         remove_file('.rtd-environment.yml')
         remove_file('readthedocs.yml')
 
-    if '{{ cookiecutter.include_cextern_folder }}' != 'y':
-        remove_dir("cextern")
-
     if '{{ cookiecutter.include_example_code }}' != 'y':
         remove_dir('{{ cookiecutter.module_name }}/example_subpkg/')
         remove_file('{{ cookiecutter.module_name }}/example_mod.py')
         remove_file('{{ cookiecutter.module_name }}/tests/test_example.py')
 
-    if '{{ cookiecutter.include_example_cython_code }}' != 'y':
+    if '{{ cookiecutter.use_compiled_extensions }}' != 'y' or '{{ cookiecutter.include_example_code }}' != 'y':
         remove_file('{{ cookiecutter.module_name }}/example_c.pyx')
 
     if '{{ cookiecutter.initialize_git_repo }}' == 'y':
```

Comment:
Do we want to keep this? I pulled it from the OA one, feels more like a documentation problem now.

Reply:
I want to remove it but in a separate PR to keep things from escalating too much

Reply:
does it trigger a bunch of doc changes or something? it's easy to remove from this angle.

Reply:
I decided to remove it in the end